packages/flutter_tools/test/integration.shard/template_manifest_test.dart

```diff
@@ -0,0 +1,31 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io'; // ignore: dart_io_import
+
+import 'package:path/path.dart' as path; // ignore: package_path_import
+import 'package:flutter_tools/src/convert.dart';
+import '../src/common.dart';
+
+/// Checks that all active template files are defined in the template_manifest.json
+void main() {
+  test('Check template manifest is up to date', () {
+    final Map<String, Object> manifest = json.decode(
+      File('templates/template_manifest.json').readAsStringSync(),
+    ) as Map<String, Object>;
+    final Set<Uri> declaredFileList = Set<Uri>.from(
+      (manifest['files'] as List<Object>).cast<String>().map<Uri>(path.toUri));
+
+    final Set<Uri> activeTemplateList = Directory('templates')
+      .listSync(recursive: true)
+      .whereType<File>()
+      .where((File file) => path.basename(file.path) != 'template_manifest.json')
```

Comment:
For those of us on Macs, can you also ignore `.DS_Store`?
```
00:03 +0 -1: Check template manifest is up to date [E]                                                                                                                              
  Expected: empty
    Actual: Set:[
              Uri:templates/.DS_Store,
              Uri:templates/app/ios-swift.tmpl/.DS_Store,
              Uri:templates/module/ios/library/.DS_Store
            ]
```

Reply:
Noooo

Reply:
I always forget about .DS_Store